tests/e2e/accounts/accounts_users/test_sync_accounts_users.py

```
@@ -0,0 +1,191 @@
+
+@pytest.fixture
+def created_account_user(mpt_vendor, account_user_factory, account_id):
+    """Fixture to create and teardown an account user."""
+    ret_account_user = None
+
+    def _created_account_user(  # noqa: WPS430
+        first_name: str = "E2E Created",
+        last_name: str = "Account User",
+    ):
+        """Create an account user with the given first and last name."""
+        nonlocal ret_account_user  # noqa: WPS420
```

Ruff (0.14.6)

Unused `noqa` directive (unknown: `WPS420`)

Remove unused `noqa` directive

(RUF100)